tests/output/InteractionsTest.py

```diff
@@ -4,15 +4,17 @@
 
 from coalib.output.Interactions import fail_acquire_settings
 from coalib.output.printers.LogPrinter import LogPrinter
+from coalib.settings.Section import Section
 
 
 class InteractionsTest(unittest.TestCase):
 
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
+        section = Section("")
         self.assertRaises(TypeError, fail_acquire_settings, log_printer, None)
```

Comment:
If I delete the line `514` mentioned above, I'm getting this error `Warning, treated as error:
/home/ubuntu/coala/coalib/output/ConsoleInteraction.py:docstring of coalib.output.ConsoleInteraction.acquire_settings:18: WARNING: Literal block ends without a blank line; unexpected unindent.`